int32 iteration
float32 distance_total
float32 entropy_total

string method_generation
string method_selection
geometry_msgs/Pose selected_pose
float32 selected_utility

float32 time_iteration
float32[] utilities
